Backend/src/AchievementService.ts

```diff
@@ -0,0 +1,17 @@
+import { ActualTask, ActualAchievement, Challenge } from './Model';
+
+export interface AchievementService {
```

Comment:
Looks like this is service, so `export class AchievementService {`
Please rename the file to `achievement.service.ts`